inc/classes/class-srm-post-type.php

```diff
@@ -252,6 +252,20 @@ public function action_redirect_chain_alert() {
 				</div>
 				<?php
 			}
+
+			if ( ! empty( $_GET['duplicate-redirect-rule'] ) ) {
+
+				?>
+				<div class="notice notice-error is-dismissible">
+					<p>
+						<?php esc_html_e( 'Safe Redirect Manager Error: Duplicate redirection rule found. You can modify below rule instead of adding new one.', 'safe-redirect-manager' ); ?>
+					</p>
+				</div>
+				<?php
+
+				// Delete duplicate post.
+				wp_delete_post( $_GET['duplicate-redirect-rule'], true );
```

Comment:
Before deleting the post on this passed id, we should add a nonce check, capability check (can the current user delete this?) and sanitize the $_GET variable for security. Otherwise this could be abused.